src/gluonts/ev/metrics.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from typing import Collection, Optional, Union, Callable
+
+from dataclasses import dataclass
+
+import numpy as np
+
+from .api import PointMetric, LocalMetric
+
+
+# --- POINT METRICS ---
+
+
+class AbsTarget(PointMetric):
+    name: str = "abs_target"
+
+    def get(self, input_data, label, forecast, metrics) -> np.ndarray:
+        return np.abs(label["target"])
+
+
+class Error(PointMetric):
+    name: str = "error"
+
+    def get(self, input_data, label, forecast, metrics) -> np.ndarray:
+        return label["target"] - forecast.median
```

Comment:
That is a good point that I neglected so far... One thing that comes to mind is a naming convention like error_q90, error_q50, error_mean (which could be confused with taking the mean of multiple error values...) but it seems brittle. With parameters, caching might not work as easily, but it might be the way to go. 👍